tests/test_office_structure/test_add_node.py

```diff
@@ -0,0 +1,43 @@
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.office_structure.create_node_fixtures import (
+    node_creation_mutation,
+    duplicate_node_creation,
+    child_node_creation_mutation_with_non_existent_parent_id
+)
+
+
+class TestAddNode(BaseTestCase):
+
+    def test_add_node(self):
+        """
+        Tests an admin can node create a node
```

Comment:
Kindly pay more attention to your typo errors

Reply:
Sorry for this. This has been fixed